lib/src/selectors/date_picker.dart

```diff
@@ -53,12 +54,18 @@ class MacosDatePicker extends StatefulWidget {
   /// {macro onDateChanged}
   final OnDateChanged onDateChanged;
 
+  /// Set an intial date for the picker.
+  /// Defaults to `DateTime.now()`.
+  final DateTime? initialDate;
+
   @override
   State<MacosDatePicker> createState() => _MacosDatePickerState();
 }
 
 class _MacosDatePickerState extends State<MacosDatePicker> {
-  final _initialDate = DateTime.now();
+  final _today = DateTime.now();
+  late final _initialDate = widget.initialDate ?? _today;
```

Comment:
Is there a reason that `_initialDate` needs to be `late`?

Reply:
Yes, `widget` and `_today` can't be accessed in an initializer, but can be with `late` applied. Another way to write it is with an initialization happening in `initState`